pkg/codegen/schema/schema.go

```diff
@@ -190,6 +199,8 @@ type Property struct {
 	Comment string
 	// Type is the type of the property.
 	Type Type
+	// ConstValue is the constant value for the property, if any.
+	ConstValue *ConstValue
```

Comment:
```suggestion
	ConstantValue interface{}
```

Reply:
Because this is part of the higher-level model, we can be more free with future breaking changes, so we don't need the single-field struct type.